node/heartbeat/proto/heartbeat.proto

```diff
@@ -18,21 +16,21 @@ message Heartbeat {
 
 // HeartbeatDTO is the struct used for handling DB operations for heartbeatMessageInfo struct
 message HeartbeatDTO {
-	google.protobuf.Duration  MaxDurationPeerUnresponsive  = 1 ;
-	google.protobuf.Duration  MaxInactiveTime              = 2 ;
-	google.protobuf.Duration  TotalUpTime                  = 3 ;
-	google.protobuf.Duration  TotalDownTime                = 4 ;
-	google.protobuf.Timestamp TimeStamp                    = 5 ;
-	bool                      IsActive                     = 6 ;
-	uint32                    ReceivedShardID              = 7 ;
-	uint32                    ComputedShardID              = 8 ;
-	string                    VersionNumber                = 9 ;
-	string                    NodeDisplayName              = 10;
-	bool                      IsValidator                  = 11;
-	google.protobuf.Timestamp LastUptimeDowntime           = 12;
-	google.protobuf.Timestamp GenesisTime                  = 13;
+	int64  	MaxDurationPeerUnresponsive  = 1 ;
+	int64  	MaxInactiveTime              = 2 ;
+	int64  	TotalUpTime                  = 3 ;
+	int64  	TotalDownTime                = 4 ;
+	int64 	TimeStamp                    = 5 ;
+	bool    IsActive                     = 6 ;
+	uint32  ReceivedShardID              = 7 ;
+	uint32  ComputedShardID              = 8 ;
+	string  VersionNumber                = 9 ;
+	string  NodeDisplayName              = 10;
+	bool    IsValidator                  = 11;
+	int64	LastUptimeDowntime           = 12;
+	int64   GenesisTime                  = 13;
 }
 
 message DbTimeStamp {
-	google.protobuf.Timestamp TS                    = 1 ;
-}
+	int64 Timestamp = 1;
+}
```

Comment:
newline characters

Reply:
ok, regarding proxy, will launch a system test and will check. Expected to work accordingly as these structures are use only on local storage.

Reply:
add new line to end of file.

Reply:
done